get-element/getElement.js

```diff
@@ -0,0 +1,40 @@
+/**
+    Returns a promise that resolves with an object containing all elements that match a given CSS selector. If no elements are found, it will wait for mutations on the document body and retry until elements matching the selector are found or a timeout is reached.
+    @param {string} cssSelector - The CSS selector used to select elements to be found.
+    @param {number} [outTimer=10000] - The maximum time in milliseconds to wait for elements to be found. Defaults to 100000ms.
+    @returns {Promise} - A promise that resolves with an object containing the CSS selector and an array of found elements.
+    @throws {Error} - If a timeout is reached before elements matching the selector are found.
+    */
+const getElement = (cssSelector, outTimer = 10000) => {
```

Comment:
Could “outTimer” be renamed “msTimeout”?

Reply:
How about calling it just "timeout"? it will then be consistent with the other functions

Reply:
@wilsonweb 
This has now been addressed in the latest commit